### ISSUE TITLE
Protractor failed to start firefox latest version - Accessibility support partially disabled

### ISSUE THREAD

**Issue by arjunsinghk3** [angular/protractor#3980]
I am trying to run protractor with Firefox latest version. It failed to start Firefox, showing the following warning.

"Accessibility support is partially disabled due to compatibility issues with new Firefox features"

**Comment by mgiambalvo** [comment]
Protractor currently only supports Firefox 47, though it can be made to work with Firefox 51 (see https://github.com/angular/protractor/pull/3953) using directConnect. Unfortunately, Firefox 51 will not work using Selenium Standalone 3.0.1 - this is due to a fixed bug (https://github.com/SeleniumHQ/selenium/issues/3115), so we're waiting on the next Selenium release.

Sorry for the inconvience. Once Firefox 51 and the next version of Selenium are released, things will be better.

**Comment by mgiambalvo** [comment]
Closing as duplicate of #3823
